### DISCUSSION TITLE
WindiCSS support?

### DISCUSSION THREAD

**Discussion by endigma** [RyanClementsHax/tailwindcss-themer#23] [Ideas]
https://windicss.org/plugins/interfaces.html
https://windicss.org/plugins/transform.html

**Comment by RyanClementsHax** [comment]
Thanks for the idea. For now, I'm just looking to support tailwind, but I wouldn't mind considering a pull request if someone from the community wanted to contribute. Does windicss have the same theming issue?

**Comment by endigma** [comment]
It relies on complex manual css, same as tailwind
